library/MedEx/MedEx_background.php

```diff
@@ -15,6 +16,10 @@
  *      but at least once each morning before 8AM localtime on work days.
  *    eg. every 29 minutes ==> active=1, execute_interval=29 (default installed values)
  *        four times a day ==> active=1, execute_interval=360 (60 minutes x 6)
+ *      EACH PERSON LOGGED IN WILL EXECUTE THIS FILE.
+ *  LARGE PRACTICES SHOULD DISABLE MedEx in background_services table and instead use MedEx_cron.php
+ *  It should be run from the practice server every 5 minutes as a suggested frequency to ensure medex_outgoing table
+ *  is up-to-date.
```

Comment:
If you set the execute_interval in the background services medex database entry, then it will never be run more than every five minutes. And if you don't want to miss any gaps (since the above "poor man" method requires users logged in to initiate it), then you can also execute the library/ajax/execute_background_services.php as a cron service as frequently as you want also (no matter how frequently you run it, it will never exceed the execute_interval).

Reply:
This is the way to go...